clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -164,10 +164,21 @@ register_csv.flush()
 invite_csv.write("SEQUENTIAL\n")
 for caller, callee in zip(callers, callees):
     if args.wildcards:
-        req_uri = "sip:{0}{1}9877654@{2}".format(callee, WILDCARD_DELIMITER, args.domain)
+        wildcard_callee = "sip:{}{}!.*!@{}".format(callee, WILDCARD_DELIMITER, args.domain)
+        caller = "{0}{1}45678".format(caller, WILDCARD_DELIMITER)
+        callee = "{0}{1}9877654".format(callee, WILDCARD_DELIMITER)
```

Comment:
Is there any reason for numbering the arguments on these string format line but not on the lines either side?